docs/getting_started.md

```diff
@@ -1,53 +1,156 @@
+
 ## Requirements
 
 - Kubernetes cluster (Docker Desktop with enabled Kubernetes)
-- `kubectl` installed, configured and able to communicate with kubernetes cluster. _`qliksense` CLI uses `kubectl` under the hood to perform operations on cluster_
+- `kubectl` installed, configured and able to communicate with kubernetes cluster. _`qliksense` CLI uses `kubectl` to perform some operations on cluster_
 
 ## Installing `qliksense` CLI
 
 Download the executable for your platform from [releases page](https://github.com/qlik-oss/sense-installer/releases) and rename it to `qliksense`
 
 ??? tldr "Linux"
 
-    ``` bash
-    curl -Lo qliksense https://github.com/qlik-oss/sense-installer/releases/download/v0.7.0/qliksense-linux-amd64
-    chmod +x qliksense
-    sudo mv qliksense /usr/local/bin
-    ```
+```bash
+curl -LOJ https://storage.googleapis.com/kubernetes-release/release/v1.16.8/bin/linux/amd64/kubectl
+curl -LOJ https://github.com/qlik-oss/sense-installer/releases/latest/download/qliksense-linux-amd64
+sudo mv qliksense-linux-amd64 kubectl /usr/local/bin
+sudo chmod ugo+x /usr/local/bin/qliksense-linux-amd64 /usr/local/bin/kubectl
+sudo ln -s /usr/local/bin/qliksense-linux-amd64 /usr/local/bin/qliksense
+sudo ln -s /usr/local/bin/qliksense-linux-amd64 /usr/local/bin/kubectl-qliksense
+```
 
 ??? tldr "MacOS"
 
-    ``` bash
-    curl -Lo qliksense https://github.com/qlik-oss/sense-installer/releases/download/v0.7.0/qliksense-darwin-amd64
-    chmod +x qliksense
-    sudo mv qliksense /usr/local/bin
-    ```
+```bash
+curl -LOJ https://storage.googleapis.com/kubernetes-release/release/v1.16.8/bin/darwin/amd64/kubectl
+curl -LOJ https://github.com/qlik-oss/sense-installer/releases/latest/download/qliksense-darwin-amd64
+sudo mv qliksense-darwin-amd64 kubectl /usr/local/bin
+sudo chmod ugo+x /usr/local/bin/qliksense-darwin-amd64 /usr/local/bin/kubectl
+sudo ln -s /usr/local/bin/qliksense-darwin-amd64 /usr/local/bin/qliksense
+sudo ln -s /usr/local/bin/qliksense-darwin-amd64 /usr/local/bin/kubectl-qliksense
+```
 
 ??? tldr "Windows"
-    Download Windows executable and add it in your `PATH` as `qliksense.exe`
-
-    [https://github.com/qlik-oss/sense-installer/releases/download/v0.7.0/qliksense-windows-amd64.exe](https://github.com/qlik-oss/sense-installer/releases/download/v0.7.0/qliksense-windows-amd64.exe)
-    
 
+```powershell
+Invoke-WebRequest https://storage.googleapis.com/kubernetes-release/release/v1.16.8/bin/windows/amd64/kubectl.exe -O C:\bin\kubectl.exe
+Invoke-WebRequest https://github.com/qlik-oss/sense-installer/releases/latest/download/qliksense-windows-amd64.exe -O C:\bin\qliksense.exe
+Copy-Item C:\bin\qliksense.exe C:\bin\kubectl-qliksense.exe
+# Add C:\bin to current Path
+$Env:Path += ";C:\bin"
+# Save Path to User environment scope
+[Environment]::SetEnvironmentVariable("Path",[Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::User) + ";C:\bin",[EnvironmentVariableTarget]::User)
+```

```

Comment:
```suggestion
    ``` powershell
    # powershell

    Invoke-WebRequest https://storage.googleapis.com/kubernetes-release/release/v1.16.8/bin/windows/amd64/kubectl.exe -O C:\bin\kubectl.exe
    Invoke-WebRequest https://github.com/qlik-oss/sense-installer/releases/latest/download/qliksense-windows-amd64.exe -O C:\bin\qliksense.exe
    Copy-Item C:\bin\qliksense.exe C:\bin\kubectl-qliksense.exe
    # Add C:\bin to current Path
    $Env:Path += ";C:\bin"
    # Save Path to User environment scope
    [Environment]::SetEnvironmentVariable("Path",[Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::User) + ";C:\bin",[EnvironmentVariableTarget]::User)
    ```
```

Reply:
Indentation